make_sleuth_folders.py

```diff
@@ -1,61 +1,76 @@
+"""This utility creates directories for the Sleuth files for University
+of London's 'Intro to Programming I' course.
+
+Run the program in the folder where you want to create the directories.
+You can create all the directories, or just the rookie/pro stages."""
+
 import os
 
 # get current directory
-currentPath = os.getcwd()
+CURRENT_PATH = os.getcwd()
+SUB_PATHS = [1, 2, 3, 4]
+
+
+def create_structure():
+    """Create directories and sub-directories required for rookie and
+    pro stages of Sleuth cases."""
 
-print("Files are created in the working directory.")
-print("Which folders do you want to create?" )
+    print("Files are created in the working directory.")
+    print("Which folders do you want to create?")
+
+    # prompt user for input and repeat if invalid
+    answer = None
+    valid_inputs = ["a", "r", "p", "x"]
+    while answer not in valid_inputs:
+        answer = input("(A)ll (R)ookie (P)ro, or e(X)it: ")
+        answer = answer.lower()
+
+    if answer == "a":
+        create_rookie()
+        create_pro()
+        print("Complete")
+    elif answer == "r":
+        create_rookie()
+        print("Complete")
+    elif answer == "p":
+        create_pro()
+        print("Complete")
+    else:
+        exit()
 
-# prompt user for input and repeat if invalid
-answer = 0
-validInputs = ["a", "r", "p", "x"]
-while answer not in validInputs:
-	answer = input("(A)ll (R)ookie (P)ro, or e(X)it: ")
-	answer = answer.lower()
 
-# Create rookie folder function
 def create_rookie():
+    """Create rookie folder function."""

```

Comment:
Comments about functions should preferably appear inside the function itself, so if you reuse a generic function in another script, it will already be documented properly.